documentation/source/diagrams/xml/hal_cosimulation.xml

```diff
@@ -1 +1 @@
-<mxfile userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36" type="device"><diagram>1Zhfb+I4EMA/DdLeQ1eQhLR9BArtSlepOrrq3qMvNsSSiZExBb59x8k4/8wG9gKU8gKe4InnN+OZsTv+aLF9VGQZP0vKRMfr0m3Hf+h4ntfvduHLSHaZ5Db0M8FccZqJeihYc8pWFZGWUmi+rAojmSQs0hXZTIqqsiWZM0cwjYhwpW+c6jiT3nlhIX9ifB7b1/TC++zJSu+sDspmZC30TSqCZ+bxglhdqd3+GNgoKUGN+bXYjpgwfKzpGZzJb57mi1QswYU0TwgQ9jsRa1yks+pNzDWbLklkxhtwWscfUrKKmdGBb3xnSjN04Z5FpCJcwSOTC6bVDv6CE/p3uAr0eM+GwKYA3QtRFpcg58sn6PB5rruwGX6g2fsR+OinT0UQWJObEFhMp0dwewUIbjBUGxHcng1B/woQ9PqHEUCCPBMCz42C6duPRweDkuuEos3+cB+UGRdiJIVU6QR/MhmNxiHIVzGhcoMT54pQDmAqfzSfU4C8r4G0fA7Fkp3XiqMbSq9spdtzHIeG5EU5HoHRxmgZY56qWnHE0lri+DT4+xThOA4vjLEWjt6xqc0/BcY7d1vHfAESwf9TBJV98Q0OXeKRRDHJtSLqdgwPP1/bYwSIfaBzQYxhre3wQsxdZYy2Lpcx2m6sVcl1e88B/JbJzfNzx2x+f0vA/gfFwcavmD0Dmy5zusH3S/K1LUlDS8MSOlAqpREJslrxCJiAeWr3C0RdO/jXDGDtZkgnwLVoeBh1zk01PvBCuVapZ0pe10TNmW1f0g3lclRMEA3eryhrhQQJlJCwrWYqIeLbXy4cMG2Kw0Qm8DVkW64NGRM65ncGBip+E7X/xcl2ERVOafJyOR0sx1Z4NFB8x4vksMQi61rnYUAH9TKfGYCzysfOmqI8ClCRX1eUWe0oSv2bG36cy/H2oORyxQg17p5sFGSNfY5v3hW4EayHT7ErbK9T8XZauC+xKwIH0WydRJrL5M/hnCFl2A6mDCcLoEvAcbtQmFNjAjUC1jUkgs8TwwWsgZLlD0354HCHNMAHC06pmTM8QXXJj6wNh8V9NwYnOS27PeXLTscyuQ4yh0/R5wITuDX3isAER9wvnI2MW3r/eXVPc5+BJbxcwMCwuNzNqllxC+6PPwA=</diagram></mxfile>
\ No newline at end of file
+<mxfile userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.132 Safari/537.36" type="device"><diagram>1Zhfb+I4EMA/DdLeQ1eQhLR9BArtSlepOrrq3qMvNsSSiZExBb59x8k4/8wG9gKU8gKe4InnN+OZsTv+aLF9VGQZP0vKRMfr0m3Hf+h4ntfvduHLSHaZ5Db0M8FccZqJeihYc8pWFZGWUmi+rAojmSQs0hXZTIqqsiWZM0cwjYhwpW+c6jiT3nlhIX9ifB7b1/TC++zJSu+sDspmZC30TSqCZ+bxglhdqd3+GNgoKUGN+bXYjpgwfKzpGZzJb57mi1QswYU0TwgQ9jsRa1yks+pNzDWbLklkxhtwWscfUrKKmdGBb3xnSjN04Z5FpCJcwSOTC6bVDv6CE/p3uAr0eM+GwKYA3QtRFpcg58sn6PB5rruwGX6g2fsR+OinT0UQWJObEFhMp0dwewUIbjBUGxHcng1B/woQ9PqHEUCCPBMCz42C6duPRweDkuuEos3+cB+UGRdiJIVU6QR/MhmNxiHIVzGhcoMT54pQDmAqfzSfU4C8r4G0fA7Fkp3XiqMbSq9spdtzHIeG5EU5HoHRxmgZY56qWnHE0lri+DT4+xThOA4vjLEWjt6xqc0/BcY7d1vHfAESwf9TBJV98Q0OXeKRRDHJtSLqdgwPP1/bYwSIfaBzQYxhre3wQsxdZYy2Lpcx2m6sVcl1e88B/JbJzfNzx2x+f0vA/gfFwcavmD0Dmy5zusH3S/K1LUlDS8MSOlAqpREJslrxCJiAeWr3C0RdO/jXDGDtZkgnwLVoeBh1zk01PvBCuVapZ0pe10TNmW1f0g3lclRMEA3eryhrhQQJlJCwrWYqIeLbXy4cMG2Kw0Qm8DVkW64NGRM65ncGBip+E7X/xcl2ERVOafJyOR0sx1Z4NFB8x4vksMQi61rnYUAH9TKfGYCzysfOmqI8ClCRX1eUWe0oSv2bG36cy/H2oORyxQg17p5sFGSNfY5v3hW4EayHT7ErbK9T8XZauC+xKwIH0WydRJrL5M/hnCFl2A6mDCcLoEvAcbtQmFNjAjUC1jUkgs8TwwWsgZLlD0354HCHNMAHC06pmTM8QXXJj6wNh8V9NwYnOS27PeXLTscyuQ4yh0/R5wITuDX3isAER9wvnI2MW3r/eXVPc5+BJbxcwMCwuNzNqllxC+6PPwA=</diagram></mxfile>
```

Comment:
Add this file to the exclude list as well, please. (As a general rule: All auto-generated files should be excluded.)

Reply:
The other XMLs in the same directory have newlines already.